### DISCUSSION TITLE
Conditions: Field Rule - show

### DISCUSSION THREAD

**Discussion by dgram** [directus/directus#13104] [Feature Requests]
Hey, the current workflow of conditions is pretty nice, but there is one thing missing.

It would be nice, if we can setup rules and **show** elements instead of hide them.

**Use Case:**
We have a collection marketing message, inside them there are marketing messages of different medium types (email, sms, facebook post etc.). 80% of the fields are the same across the mediums. Currently we have to setup a negiate rule.

**Example:**
When we only want to show the field "pre-description" for emails, we have to setup a filter named **isNotEmail** and hide the "pre-description" field, it would be nice to make this workflow also vice versa (**isEmail** and show the item only then).

regards

**Comment by azrikahar** [comment]
Assuming I understood correctly, you should be able to do so by making the field _hidden_ by default (set hidden to `true`), then create the isEmail condition and set hidden to `false`, which will show the field. 👍

**Comment by dgram** [comment]
shame on me, i didn't see the "types" of hidden field config.

thanks for fast response and regards

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
